.github/workflows/node-ci.yml

```diff
@@ -30,13 +30,3 @@ jobs:
       #   run: yarn cover
       - name: Build example project
         run: yarn build
-
-      # at this point, the build is successful
-      - name: Codecov
-        uses: codecov/codecov-action@v1.2.0
-        continue-on-error: true
-      - name: Semantic Release
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
There's no need to publish to npm, but it's still nice to have semver handled for us (plus release notes).